packages/evm/contracts/lib/os/ERC20.sol

```diff
@@ -1,7 +1,7 @@
 // Brought from https://github.com/aragon/aragonOS/blob/v4.3.0/contracts/lib/token/ERC20.sol
```

Comment:
Given that we're more or less jettisoning aragonOS, how do you feel about changing these to be just in `lib` or even modified to use their OZ-equivalent?

Reply:
I have just read this but I think I did this in #18